README.md

```diff
@@ -27,6 +28,18 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 1. In your browser, visit [http://127.0.0.1:8000/](http://127.0.0.1:8000/) or [http://localhost:8000/](http://localhost:8000/)
 1. Press CTRL+C in the terminal to stop the server.
 
+## For mac users
+
+1. Clone the project (`git clone https://github.com/codethesaurus/codethesaur.us.git`)
+1. Run `pip3 install virtualenv`
+1. To set up new virtual environment, run `virtualenv --no-site-packages venv`
+1. To activate virtual environment, run `source venv/bin/activate`
+1. Run `pip install -r requirements.txt`
```

Comment:
Change this to `pip3` and line 38 to `python3` and I'll merge it!

Reply:
Actually, in the virtual environment, we don't need pip3 or python3, pip or python should be okay. But if you still want this change, I can make it too. Please let me know. Thanks.

Reply:
oh yeah, good point. Alright, let's do it!